mne_realtime/lsl_client.py

```diff
@@ -70,9 +71,13 @@ def get_data_as_epoch(self, n_samples=1024, picks=None):
 
         # create an event at the start of the data collection
         events = np.expand_dims(np.array([0, 1, 1]), axis=0)
-        samples, _ = self.client.pull_chunk(max_samples=n_samples,
-                                            timeout=wait_time)
-        data = np.vstack(samples).T
+        _, timestamps = self.client.pull_chunk(
+            max_samples=self.buffer.shape[0],
+            timeout=wait_time,
+            dest_obj=self.buffer,
```

Comment:
do you know if it always fills the beginning of the buffer array?

Reply:
The `lsl` C++ bindings are a little difficult for me to follow, but the `pylsl` example code seems to assume that it fills the beginning of the buffer array: https://github.com/labstreaminglayer/pylsl/blob/master/pylsl/examples/ReceiveAndPlot.py#L69-L75

Also, the Python->C++ function call passes in the entire buffer array (without any additional indexing), so it seems like the C++ code should write to the start of the buffer array.